### PR TITLE
Added update and delete options for Book entity.

### PR DESCRIPTION
**Pull request by k0sm0naft**
Added JpaRepository.
Implemented update and delete operations.
Implemented soft-delete concept.
Added Liquibase support.

### CODE REVIEW

File: src/main/java/ua/bookstore/online/service/BookServiceImpl.java [hunk 1]
```diff
@@ -35,4 +35,16 @@ public BookDto getById(Long id) {
                 () -> new EntityNotFoundException("Entity not found by id " + id));
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public BookDto update(Long id, CreateBookRequestDto bookRequestDto) {
```
Comment (yevheniisynytsia) [inline src/main/java/ua/bookstore/online/service/BookServiceImpl.java:40]: we should at least check if we have a record with such ID.
There are two options - using `existsById()`, or getting record from db - updating all its fields and saving back

File: src/main/resources/db/changelog/changes/01-create-books-table.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (yevheniisynytsia) [inline src/main/resources/db/changelog/changes/01-create-books-table.yml]: primaryKey is not null by default

File: src/main/resources/db/changelog/changes/01-create-books-table.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                  constraints:
+                    nullable: false
+              - column:
+                  name: isbn
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: price
+                  type: decimal
```
Comment (yevheniisynytsia) [inline src/main/resources/db/changelog/changes/01-create-books-table.yml]:
```suggestion
                  type: decimal(12, 2)
```

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]

**Review by andrii-hoienko — APPROVED** [review]
Good job!

**Pull request merged**